server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/application/deployment/Deployment.java

```diff
@@ -9,48 +10,54 @@
+    private BashScript deploymentScript;
+    private Application application;
+    private PluginFileAccess fileAccess;
+    private Class deploymentClass;
 
-    private final static String PYTHON_CONFIGURE_SQL_FILENAME = "configureMysql.py";
-    private final static String PYTHON_CONFIGURE_SQL_SOURCE = "/cloudFoundry/deployment_scripts/";
+    private final String PYTHON_SCRIPTS_TARGET = SCRIPTS_DIR_PATH;
 
-    private final static String PYTHON_EXECUTE_FILENAME = "executeCommand.py";
-    private final static String PYTHON_EXECUTE_SOURCE = "/cloudFoundry/deployment_scripts/";
+    private final String PYTHON_CONFIGURE_SQL_FILENAME = "configureMysql.py";
+    private final String PYTHON_CONFIGURE_SQL_SOURCE = "/cloudFoundry/deployment_scripts/";
 
-    private final static String PYTHON_READ_CREDENTIALS_FILENAME = "readCredentials.py";
-    private final static String PYTHON_READ_CREDENTIALS_SOURCE = "/cloudFoundry/deployment_scripts/";
+    private final String PYTHON_EXECUTE_FILENAME = "executeCommand.py";
+    private final String PYTHON_EXECUTE_SOURCE = "/cloudFoundry/deployment_scripts/";
 
-    private final static String PYTHON_REPLACE_STRINGS_FILENAME = "replace.py";
-    private final static String PYTHON_REPLACE_STRINGS_SOURCE = "/cloudFoundry/deployment_scripts/";
+    private final String PYTHON_READ_CREDENTIALS_FILENAME = "readCredentials.py";
+    private final String PYTHON_READ_CREDENTIALS_SOURCE = "/cloudFoundry/deployment_scripts/";
 
-    private BashScript deploymentScript;
-    private Application application;
-    private PluginFileAccess fileAccess;
-    private Class deploymentClass;
-    private boolean pythonIsChecked;
+    private final String PYTHON_REPLACE_STRINGS_FILENAME = "replace.py";
+    private final String PYTHON_REPLACE_STRINGS_SOURCE = "/cloudFoundry/deployment_scripts/";
 
     public Deployment(BashScript deploymentScript, Application application, PluginFileAccess fileAccess) throws IOException {
         this.deploymentScript = deploymentScript;
         this.application = application;
         this.fileAccess = fileAccess;
         deploymentClass = Deployment.class;
-        this.pythonIsChecked = false;
     }
 
     /**
      look for suitable services which match to the requirements of the user
-
-     @param showAllServiceOfferings if yes in the deployment script will be all offered services added
      */
-    public void treatServices(Boolean showAllServiceOfferings) {
+    public void treatServices() throws IOException {
         ServiceHandler serviceHandler = new ServiceHandler(application, deploymentScript);
-        serviceHandler.addServiceCommands(showAllServiceOfferings);
+
+        String scriptPath = deploymentScript.getScriptPath();
+        File scriptFile = new File(scriptPath);
+        String contentScript = FileUtils.readFileToString(scriptFile);
+
+        if (!contentScript.contains("cf create-service")) {
```

Comment:
You could replace this if-else with a one-liner like:
`serviceHandler.addServiceCommands(!contentScript.contains("cf create-service"));`
It would probably make this a bit more readable.